thirdparty/junixsocket/zimbra-junixsocket/debian/control

```diff
@@ -0,0 +1,24 @@
+Source: zimbra-junixsocket
+Build-Depends: debhelper (>= 9), m4, dpkg-dev (>= 1.15.7), maven3
+Section: utils
+Priority: optional
+Maintainer: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Uploaders: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Standards-Version: 3.9.5
+Homepage:  https://github.com/kohlschutter/junixsocket
+
+Package: zimbra-junixsocket
+Section: jars
+Priority: important
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}, zimbra-base
+Description: junixsocket JARs
+
+Package: zimbra-junixsocket-dbg
+Section: debug
+Priority: extra
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}, zimbra-base
```

Comment:
the debug package should depend on the zimbra-junixsocket package, no?  Seems like wouldn't be much point to installing it w/o the base package. ;)